lib/fizzy/utf8.cpp

```diff
@@ -108,7 +109,7 @@ bool utf8_validate(const uint8_t* pos, const uint8_t* end) noexcept
 
         // At this point need to read at least one more byte.
         assert(required_bytes > 1);
-        if ((end - pos) < required_bytes - 1)
+        if ((end - pos) < (required_bytes - 1))
```

Comment:
Not sure if makes sense, but perhaps it would be simpler, to just decrement all `required_bytes` literals now, and use `required_bytes` here without subtraction.

Reply:
I don't really want to change more than needed here. I believe we should replace it with something more performant anyway in the future.